Marlin/src/pins/pins_AZTEEG_X3_PRO.h

```diff
@@ -78,7 +78,7 @@
 // Z Probe (when not Z_MIN_PIN)
 //
 #ifndef Z_MIN_PROBE_PIN
-  #define Z_MIN_PROBE_PIN   12
+  #define Z_MIN_PROBE_PIN   22
```

Comment:
Are these in conflict with the changes Roo made recently, or was this intentional?